ee-9/pom.xml

```diff
@@ -2784,6 +2784,90 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.ws.cxf</groupId>
+                <artifactId>jbossws-cxf-client</artifactId>
+                <version>${preview.version.org.jboss.ws.cxf}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.jboss.ws.cxf</groupId>
+                <artifactId>jbossws-cxf-factories</artifactId>
+                <version>${preview.version.org.jboss.ws.cxf}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.jboss.ws.cxf</groupId>
+                <artifactId>jbossws-cxf-jaspi</artifactId>
+                <version>${preview.version.org.jboss.ws.cxf}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.jboss.ws.cxf</groupId>
+                <artifactId>jbossws-cxf-resources</artifactId>
+                <version>${preview.version.org.jboss.ws.cxf}</version>
+                <classifier>wildfly2700</classifier>
+            </dependency>
```

Comment:
Should not this artifact exclude all the transitive dependencies?

We are now following such a pattern.

Reply:
That's a good catch Yeray. I'd agree here.